packages/uswds-core/src/styles/settings/_settings-typography.scss

```diff
@@ -418,3 +418,7 @@ $theme-lead-font-family: "heading" !default;
 $theme-lead-font-size: "lg" !default;
 $theme-lead-line-height: 6 !default;
 $theme-lead-measure: 6 !default;
+
+// Margin
+$theme-heading-margin-top: 1.5em !default;
+$theme-paragraph-margin-top: 1em !default;
```

Comment:
Note: For reference, updating this value (with all global styles set to false) updated the margins in the following items:
- `.usa-display + *`
- `* + .usa-paragraph`
- `.usa-paragraph + *`
- `.usa-prose > p + *`
- `.usa-prose > h1 + *` (through h6)
- `.usa-graphic-list__heading + *`
- `.usa-hero__heading + *`

Reply:
@amyleadem all global styles set to false is the default are you saying this is a regression? I wasn't able to replicate.

Reply:
No, I am not saying there is a regression here. My aim with this note was to add clarity about what this effectively updates, but a few weeks later I see that the comments are just confusing haha. It was just me trying to wrap my head around it. These comments can be safely ignored. Going to go ahead and strike and resolve my more confusing comments.